include/openssl/cast.h

```diff
@@ -1,96 +0,0 @@
-/* Copyright (C) 1995-1998 Eric Young (eay@cryptsoft.com)
```

Comment:
Should we keep the header file and just make it a "dummy" file with no functions? This might keep compatibility with open sources that still reference the `cast.h` header file.